meetings/2019-01-23.md

```diff
@@ -0,0 +1,67 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-01-23
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=qamraECS-EM
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/655
+
+## Present
+
+* Anatoli Papirovski @apapirovski (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Rich Trott @Trott (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* repl: emit uncaughtException [#20803](https://github.com/nodejs/node/pull/20803)
+  * Untagged from agenda.  For awareness needs TSC members to review to make progress.
+
+### nodejs/TSC
+
+* NODE_MODULE_VERSION for Electron Major Releases [#651](https://github.com/nodejs/TSC/issues/651)
+  * Issue is with projects that bundle Node.js but use different versions of the
+    dependencies (for example V8 or OpenSSL)
+  * Gabriel - single number poses some issues as it means that more versions of the
+    binaries.
+  * Michael, in terms of unbreaking 651 would it make sense to allow new version to be
+    grabbed and then figure out longer term approach.
+  * Gabriel, making it easier to grab a number is good anyway so we can start with that.
+  * Discussion in the meeting is that we just need a PR to bump the ABI number in master
+    but Gabriel ‘volunteered’ to confirm process and comment in 651 on what specifically to
+    do.
+  * Gabriel will also create a new issue to continue discussion around whether we want
+    to assign bits etc. (PoC is in https://github.com/nodejs/node/pull/25539)
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * No Myles today so no update.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * Streams, stalled waiting on people who know more about current streams implementation
+  * N-API
+    * Focus on getting tfsn non-experimental and backported in time for planned 8.x SemVer minor
```

Comment:
tfsn? Is that a typo or an acronym? If an acronym, maybe spell it out and/or link to something that explains it?

Reply:
expanded